sound/soc/intel/boards/Kconfig

```diff
@@ -554,6 +554,8 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	select SND_SOC_RT1308_SDW
 	select SND_SOC_RT1308
 	select SND_SOC_RT715_SDW
+	select SND_SOC_RT5682_SDW
+	select SND_SOC_RT5682
```

Comment:
the last line is unnecessary

````
config SND_SOC_RT5682_SDW
	tristate "Realtek RT5682 Codec - SDW"
	depends on SOUNDWIRE
	select SND_SOC_RT5682
	select REGMAP_SOUNDWIRE
````

Reply:
Agreed on Kconfig..
rebased to sof-dev and re-verified functionality..

> minor change in the Kconfig, otherwise ok once it's rebased
Agreed on Kconfig..
rebased to sof-dev and re-verified functionality..